dm/tests/all_mode/run.sh

```diff
@@ -375,6 +375,38 @@ function test_regexpr_router() {
 	echo "[$(date)] <<<<<< finish test_regexpr_router $1 >>>>>>"
 }
 
+function test_json_expression() {
```

Comment:
this function should be called at around line 414.

And I'll help to check CI if it fails